lib/rspec/rails/view_controller.rb

```diff
@@ -0,0 +1,43 @@
+module RSpec
+  module Rails
+    # Methods added to ActionView::TestCase::TestController, only in view specs
+    module ViewController
+      def self.included(_klass)
```

Comment:
@cupakromer would know better than me, but my instincts are saying before/after that adds/remove methods in a similar way to how syntax.rb works: https://github.com/rspec/rspec-expectations/blob/master/lib/rspec/expectations/syntax.rb#L38-L64